src/core/tests/CMakeLists.txt

```diff
@@ -461,7 +461,6 @@ endif()
 # TODO: Enable these tests in new infrastructure
 set(MULTI_TEST_SRC
     backend/abc.in.cpp
-    backend/interpolate.in.cpp
```

Comment:
Please remove backend/interpolate.in.cpp file

Reply:
Removed!